templates/docs-page.html

```diff
@@ -1,28 +1,18 @@
 {% extends "docs-base.html" %}
-
 {% block page_name %}
   {% if page and page.path is starting_with("/learn/book/") %}
     The Book
   {% else %}
     Migration Guides
   {% endif %}
-{% endblock %}
-
+{% endblock page_name %}
 {% block docs_page_title %}
   {% if page.extra.long_title %}
     {{ page.extra.long_title }}
   {% else %}
     {{ page.title }}
   {% endif %}
-  {% if page.extra.subtitle %}
-    <span class="docs-page-subtitle">{{ page.extra.subtitle }}</span>
-  {% endif %}
-{% endblock %}
-
-{% block docs_page_content %}
-  {{ page.content | safe }}
-{% endblock %}
-
-{% block docs_page_relative_path %}
-{{page.relative_path}}
-{% endblock %}
+  {% if page.extra.subtitle %}<span class="docs-page-subtitle">{{ page.extra.subtitle }}</span>{% endif %}
+{% endblock page_title %}
+{% block docs_page_content %}{{ page.content | safe }}{% endblock docs_page_content %}
```

Comment:
Would it be possible for the formatter to keep this multi line?

Reply:
The formatter was just a once run thing to fix the indentation after stuff had messed it up; only a linter (which doesn't actually care about indentation) will be run in CI. So we'll have to manually keep the formatting of the indentation ourselves.  I'll manually fix this.